vendor/engines/secure_rooms/spec/models/secure_rooms/indala_ability_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe SecureRooms::IndalaAbility do
+  subject(:ability) { described_class.new(current_user, facility) }
+  let(:indala_user) { FactoryGirl.create(:user) }
```

Comment:
Can we get away with `build` or `build_stubbed` for the factories in this test? I like to start with one of the non-db builders until it turns out that it's trying to hit the DB for some reason or relations break.